### ISSUE TITLE
keep same size of search field if i resize the window

### ISSUE THREAD

**Issue by CennoxX** [peter-frentrup/NppMenuSearch#37]
If I resize the Notepad++ window the size of the search field changes from my set size in the maximized window to a smaller search field:
![output](https://user-images.githubusercontent.com/6975881/143310182-1e711417-c312-4bff-b716-6a260a20fb85.gif)

A related problem is that if I close the now un-maximized window with the shrunken search field and then reopen Notepad++, the search field remains shrunk even if I maximize the window again. This might be related to issue #8.
![close](https://user-images.githubusercontent.com/6975881/143311943-15d3f817-612c-46ac-a0c9-f7c6c668eee9.gif)

**Comment by Gitoffthelawn** [comment]
I can confirm this issue.

**Comment by peter-frentrup** [comment]
Hello,
thank you for your report. It seems that you have installed the CustomizeToolbarPlugin. IIRC, last time I checked it worked by adding invisible buttons to the primary toolbar or some similar trick to grab space. That would explain why the search field shrinks when you restore the window size from maximized state (first animation). This issue is an example of two toolbars fighting for available space, so there exists probably no good fix.

For the other issue (second animation): the search field just recognized that its width was changed, but it does not know whether it was by the user actively grabbing the toolbar handle or by Windows realigning things upon window resize. When reopening, it just tries to restore the last seen width.

Sorry,
Peter

**Comment by Gitoffthelawn** [comment]
Thanks for you explanation Peter.

One possible solution would be to merge this helpful plugin with the CustomizeToolbar plugin.  That way there wouldn't be two toolbars fighting for available space.  I'm not sure if that interests you at all, but it's an idea.  Either that, or maybe the two separate plugins could communicate with each other.

Personally, I find both plugins to be very helpful.  In the end, I found the CustomizeToolbar plugin to be slightly more valuable, so I uninstalled this plugin.  I certainly miss it, but having to manually adjust the size and position got to be a bit too much of a hassle for my taste.

**Comment by CennoxX** [comment]
You're right, I use the CustomizeToolbarPlugin. But even without CustomizeToolbarPlugin the field doesn't keep the same size:
![output](https://user-images.githubusercontent.com/6975881/150696295-d1c012b3-56d6-475a-a2bd-fbfd80649eed.gif)

**Comment by peter-frentrup** [comment]
The resizing logic is how the [Windows Rebar control](https://learn.microsoft.com/en-us/windows/win32/controls/rebar-controls) works. There is not a lot I can do.

Sorry,
Peter

**Comment by peter-frentrup** [comment]
The new version 0.9.7 (binaries on [SourceForge](https://sourceforge.net/projects/nppmenusearch/)) allows to lock the search widget size via a new `⁞` button on the right.
